2002-09-09  Richard Henderson  <rth@redhat.com>

	* elfxx-ia64.c (USE_BRL): Define.

--- binutils-2.13.90.0.16/bfd/elfxx-ia64.c	9 Sep 2002 17:34:07 -0000	1.58
+++ binutils-2.13.90.0.16/bfd/elfxx-ia64.c	9 Sep 2002 17:37:25 -0000
@@ -645,7 +645,7 @@ static const bfd_byte plt_full_entry[PLT
 
 /* Select out of range branch fixup type.  Note that Itanium does
    not support brl, and so it gets emulated by the kernel.  */
-#undef USE_BRL
+#define USE_BRL
 
 static const bfd_byte oor_brl[16] =
 {
